Mathlib/Mathport/Syntax.lean

```diff
@@ -163,6 +163,11 @@ open Lean Parser.Tactic
 /- S -/ syntax (name := elide) "elide " num (location)? : tactic
 /- S -/ syntax (name := unelide) "unelide" (location)? : tactic
 
+/-- `ext1? pat*` is like `ext1 pat*` but gives a suggestion on what pattern to use -/
+/- M -/ syntax (name := ext1?) "ext1?" (colGt ppSpace rintroPat)* : tactic
+/-- `ext? pat*` is like `ext pat*` but gives a suggestion on what pattern to use -/
```

Comment:
I don't think this was ever true? I thought `ext?` in lean3 meant `show_term ext`?

Reply:
I think both descriptions are slightly inaccurate. The doc says this:
````quote
In the previous example, applying `ext? ⟨a,b⟩` will produce the trace message:

```lean
Try this: apply funext, rintro ⟨a, b⟩
```
````
which is to say, it does suggest patterns but will also intersperse them with other tactic calls.